shared/src/validation-schemas/chat-message/chat-message-create-params.validation-schema.ts

```diff
@@ -1,15 +1,23 @@
 import * as Joi from 'joi';
 
-import { ChatValidationMessage } from '~/common/enums/enums';
+import {
+  ChatValidationMessage,
+  ChatValidationRule,
+} from '~/common/enums/enums';
 import { ChatMessageCreateRequestBodyDto } from '~/common/types/types';
 import { getNameOf } from '~/helpers/helpers';
 
 const chatMessageCreateArguments = Joi.object({
   [getNameOf<ChatMessageCreateRequestBodyDto>('message')]: Joi.string()
+    .trim()
```

Comment:
Looks like we use this schema only for backend side but we need this only for UI. I mean you should remove these rules from this schema and add them to mobile schema that we use for ui. Also we have decided to set only min length for now